src/test/unit/controls/PairedResult/PairedResultLoad-spec.js

```diff
@@ -2007,5 +2007,34 @@ describe("Paired Result - Load", () => {
                     .getAttribute("aria-describedby")
             ).toEqual(pairPrimary.key);
         });
+        it("Does not affect paired results in other graphs", () => {
```

Comment:
nit: we can add a describe block here.

```js
describe("when multiple canvas with paired results present", () => {
      it("shoud not affect paired results in other canvases", () => {
```

Reply:
Added describe block [here](https://github.com/JananiGunasekaran/carbon-graphs/commit/9fd0a0d6ad845075c2a29d9795d0c699c78688fc).